pkg/server/container_create_test.go

```diff
@@ -48,8 +50,57 @@ func checkMount(t *testing.T, mounts []runtimespec.Mount, src, dest, typ string,
 	assert.True(t, found, "mount from %q to %q not found", src, dest)
 }
 
-func getCreateContainerTestData() (*runtime.ContainerConfig, *runtime.PodSandboxConfig,
-	*imagespec.ImageConfig, func(*testing.T, string, uint32, *runtimespec.Spec)) {
+func createHostPath(t *testing.T, mounts []*runtime.Mount) {
```

Comment:
Don't think we should test this in unit test.
We have replaced or `os` with `FakeOS`. This test won't work.
If we really want to test, we should leverage the `FakeOS` to test.

Reply:
I'll add our own integration test framework, we could add this kind of test there.